### ISSUE TITLE
Bug: Data Assigned to Parameters Does Not Work Correctly

### ISSUE THREAD

**Issue by dogukancim** [aoijs/aoi.js#606]
### Bug Type

Function

### Bugged Function

```js
client.functionManager.createFunction(
    {
        name: "$banLog",
        params: ["v1", "v2", "v3"],
        $if: "old",
        type: "aoi.js",
        code: `
        $if[$channelType[$getGuildVar[ban-log]]==text&&$channelExists[$getGuildVar[ban-log]]==true]
            $channelSendMessage[$getGuildVar[ban-log];
                {author:Case #$getGuildVar[incident-case] — $username[{v2}]:$userAvatar[{v2}]}
                {field:Yetkili:<@{v1}> **$username[{v1}]}
                {field:Yasaklanan:<@{v2}> **$username[{v2}]}
                {field:Sebep:{v3}}
                {timestamp:}
                {color:$getGuildVar[ui-color]}
            ;false]
        $else
            $interactionReply[<:valentine_close:1188188914276630579> Eksik veya yanlış yapılandırma tespit edildi, lütfen </config ban:> ayarına girdiğiniz verinin doğruluğunu kontrol edin. Teknik bir sorun olduğunu düşünüyorsanız lütfen </fix:> komutunu kullanın.;everyone;true;false]
        $endif
        `
    }
);
```

### Version of aoi.js

v6.7.1

### Version of Node.js

v20.13.1

### Excepted Output

The data received by my second parameter, which I assigned as v2, works incorrectly.

### Relevant log output

```shell
AoiError: Invalid User ID [{v2}] 
{ 
   Function : `$username`,
   Command : `$banLog`,
   Version : 6.7.1 
}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ddodogames** [comment]
custom functions in aoi.js type are broken, this is also made clear in the documentation itself
![image](https://github.com/aoijs/aoi.js/assets/69091682/83f27639-50b3-4e78-96ca-243438662c2d)

**Comment by dogukancim** [comment]
> custom functions in aoi.js type are broken, this is also made clear in the documentation itself ![image](https://private-user-images.githubusercontent.com/69091682/335796592-83f27639-50b3-4e78-96ca-243438662c2d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcyNDA0NTUsIm5iZiI6MTcxNzI0MDE1NSwicGF0aCI6Ii82OTA5MTY4Mi8zMzU3OTY1OTItODNmMjc2MzktNTBiMy00ZTc4LTk2Y2EtMjQzNDM4NjYyYzJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjAxVDExMDkxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkN2YyM2JhZDBhZmRjMTliM2I5MGE0MGYyM2Q5ZWIwNjkwOGE4NzgwZDA4ZWEwNDQ3OWE4MzkyMDQzOGQyMzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pWEVoSP3DDIlrZJfGKWqDOFIRU4muZQWuyWbyqV1gjc)

I didn't see that part, but I just wanted it fixed. It's bad for a really useful feature to work incorrectly.

**Comment by Faf4a** [comment]
not planned to fix until 6.9
